frontend/packages/helm-plugin/src/components/forms/HelmChartRepository/CreateHelmChartRepositoryForm.tsx

```diff
@@ -72,13 +72,21 @@ const CreateHelmChartRepositoryForm: React.FC<FormikProps<FormikValues> & {
 
   const formTitle =
     !showScopeType && formData.scope === ProjectHelmChartRepositoryModel.kind
-      ? t('helm-plugin~Create ProjectHelmChartRepository')
-      : t('helm-plugin~Create Helm Chart Repository');
+      ? t('helm-plugin~{{formType}} ProjectHelmChartRepository', {
+          formType: existingRepo ? 'Edit' : 'Create',
+        })
+      : t('helm-plugin~{{formType}} Helm Chart Repository', {
+          formType: existingRepo ? 'Edit' : 'Create',
+        });
 
   const formDescription =
     !showScopeType && formData.scope === ProjectHelmChartRepositoryModel.kind
-      ? t('helm-plugin~Add helm chart repository in the namespace.')
-      : t('helm-plugin~Add helm chart repository');
+      ? t('helm-plugin~{{formDescType}} helm chart repository in the namespace.', {
+          formDescType: existingRepo ? 'Update' : 'Add',
+        })
+      : t('helm-plugin~{{formDescType}} helm chart repository.', {
+          formDescType: existingRepo ? 'Update' : 'Add',
+        });
```

Comment:
I understand the benefit for a developer of concatenating strings like `Create`/`Edit` plus `ProjectHelmChartRepository`/`Helm Chart Repository` - or `Add`/`Update` plus a sentence.

The problem here is that this expects that all translations have the form `$verb $noun` or `$verb sentence`, but that's not required in all languages and I would expect that makes the life of the translation team more difficult.

Using variables here also will not allow the translation team to translate "Update" and "Add".

So I recommend a much more simple (you can also call it stupid) code like this:

```suggestion
  const formDescription =
    !showScopeType && formData.scope === ProjectHelmChartRepositoryModel.kind
      ? (existingRepo
          ? t('helm-plugin~Update helm chart repository in the namespace.')
          : t('helm-plugin~Add helm chart repository in the namespace.')
        )
      : (existingRepo
          ? t('helm-plugin~Update helm chart repository.')
          : t('helm-plugin~Add helm chart repository.')
        );
```